ui/src/common/layouts/CollectionsMenu/CollectionsMenu.jsx

```diff
@@ -68,12 +70,35 @@ function CollectionsMenu({ currentPathname }) {
         </CollectionLink>
       </Col>
       <Col>
-        <CollectionLink
-          active={activeCollection === INSTITUTIONS_PID_TYPE}
-          to={INSTITUTIONS}
+        <Menu
+          className="__CollectionsDropdownMenu__ mh4 m-mh2"
+          theme="dark"
+          mode="horizontal"
+          selectable={false}
+          inlineIndent={0}
         >
-          Institutions
-        </CollectionLink>
+          <Menu.SubMenu key="More" title="More...">
+            <Menu.Item key="more.institutions">
+              <CollectionLink
+                active={activeCollection === INSTITUTIONS_PID_TYPE}
+                to={INSTITUTIONS}
+                isDropdownLink
+              >
+                Institutions
+              </CollectionLink>
+            </Menu.Item>
+            <Menu.Item key="more.experiments">
+              <CollectionLink
```

Comment:
These don't need to use `CollectionLink`, just a normal `Link` would be enough since we don't need any logic or customization of `CollectionLink` here.

Reply:
I asked Stella and they need similar customisation as the collection link (eg. the highlight when they are active or new tag) so that's why I used collection link. If I change to link I'll have to duplicate this code.